### PR TITLE
Быстров - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by dtaleme**

### CODE REVIEW

File: modules/complex-number/test/test_bystrov_andrey_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+TEST(BYSTROV_ANDREY_COMPLEX_NUMBER_TEST, CopyConstructor) {
+  ASSERT_NO_THROW(ComplexNumber(ComplexNumber()));
+}
+
+TEST(BYSTROV_ANDREY_COMPLEX_NUMBER_TEST, AssignmentOperator) {
+  auto origin = ComplexNumber(1.0, 2.0);
+  auto copy = ComplexNumber();
+  ASSERT_NO_THROW(copy = origin);
```
Comment (dkurt) [inline modules/complex-number/test/test_bystrov_andrey_complex_number.cpp:24]: Check the values after assignment

File: modules/complex-number/test/test_bystrov_andrey_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 Bystrov Andrey
```
Comment (dkurt) [inline modules/complex-number/test/test_bystrov_andrey_complex_number.cpp:1]: No tests for getters and setters (getRe, setRe, getIm, setIm)

File: modules/complex-number/test/test_bystrov_andrey_complex_number.cpp (hunk 1 above)
Comment (dtaleme) [reply modules/complex-number/test/test_bystrov_andrey_complex_number.cpp:25]: Done

File: modules/complex-number/test/test_bystrov_andrey_complex_number.cpp (hunk 2 above)
Comment (dtaleme) [reply modules/complex-number/test/test_bystrov_andrey_complex_number.cpp:1]: Done

**Comment by dtaleme** [comment]
@dkurt

**Pull request merged**